src/chat/ContactSheet.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  ScreenHeaderTexts,
+  ContactSheetTexts,
+  useTranslation,
+} from '@atb/translations';
+import React, {forwardRef} from 'react';
+import {Linking, View} from 'react-native';
+import FullScreenFooter from '@atb/components/screen-footer/full-footer';
+import {ScreenHeaderWithoutNavigation} from '@atb/components/screen-header';
+import {BottomSheetContainer} from '@atb/components/bottom-sheet';
+import {Support, Chat, ChatUnread} from '@atb/assets/svg/icons/actions';
+import useChatUnreadCount from './use-chat-unread-count';
+import Intercom from 'react-native-intercom';
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+
+type Props = {
+  close: () => void;
+};
+
+const ContactSheet = forwardRef<View, Props>(({close}, focusRef) => {
+  const styles = useStyles();
+  const {t} = useTranslation();
+  const unreadCount = useChatUnreadCount();
+  const {customer_service_url} = useRemoteConfig();
+
+  return (
+    <BottomSheetContainer>
+      <ScreenHeaderWithoutNavigation
+        title={t(ContactSheetTexts.header.title)}
+        rightButton={{
+          type: 'skip',
+          onPress: close,
+          text: t(ScreenHeaderTexts.headerButton.cancel.text),
+        }}
+        color={'background_2'}
+        setFocusOnLoad={false}
+      />
+
+      <FullScreenFooter>
+        <View style={styles.descriptionSection} ref={focusRef}>
+          <ThemeText type="body__secondary" color="secondary">
+            {t(ContactSheetTexts.customer_service.title)}
+          </ThemeText>
+          <ThemeText>{t(ContactSheetTexts.customer_service.body)}</ThemeText>
+        </View>
+        <Button
+          color="primary_2"
+          text={t(ContactSheetTexts.customer_service.button)}
+          accessibilityHint={t(ContactSheetTexts.customer_service.a11yHint)}
+          onPress={() => {
+            if (Linking.canOpenURL(customer_service_url)) {
+              Linking.openURL(customer_service_url);
+            }
+            close();
+          }}
+          iconPosition="right"
+          icon={() => (
+            <ThemeIcon colorType="background_gray" svg={Support}></ThemeIcon>
+          )}
+        />
+
+        <View style={styles.descriptionSection}>
+          <ThemeText type="body__secondary" color="secondary">
+            {t(ContactSheetTexts.intercom.title)}
+          </ThemeText>
+          <ThemeText>{t(ContactSheetTexts.intercom.body)}</ThemeText>
+        </View>
+
+        <Button
+          color="primary_2"
+          text={t(ContactSheetTexts.intercom.button)}
+          accessibilityHint={t(ContactSheetTexts.intercom.a11yHint)}
+          onPress={() => {
+            unreadCount
+              ? () => Intercom.displayMessenger()
+              : Intercom.displayConversationsList();
+            close();
+          }}
+          iconPosition="right"
+          icon={() =>
+            unreadCount ? (
+              <ThemeIcon colorType="background_gray" svg={ChatUnread} />
```

Comment:
På ikonene bør `colorType` samsvare med `color` på knappen, altså at den bør være `primary_2`i stedet for `background_gray`. Både `primary_2` og `background_gray` har hvit tekst-/ikonfarge, så det ender opp riktig slik det er nå også, men best å være konsekvent.

Reply:
b679bbd5004b823de2923017f5293bdf8a6b5156 👍